src/cp/v2.c

```diff
@@ -0,0 +1,1515 @@
+// SPDX-FileCopyrightText: 2025 RizinOrg <info@rizin.re>
+// SPDX-FileCopyrightText: 2025 Siddharth Mishra <admin@brightprogrammer.in>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <stdio.h>
+
+#include "cp/common.h"
+
+
+// See issue :
+//   https://github.com/rizinorg/rz-libdemangle/issues/8
+//
+// For name mangling scheme for GNU v2 ABI, see section 8.4 (Gnu v2 name mangling)
+//   https://github.com/rizinorg/rizin/files/6154867/calling_conventions.pdf
+//
+// A better formatted document can be found here :
+//   https://kb.brightprogrammer.in/s/15fd1dd9-d47d-4ec1-9339-7c111db41ab5
```

Comment:
even tho this is ok, it would be better if this is hosted on a more global platform. i would suggest to upload the document on the github issue and link it like the one above.

Reply:
consider it done ✔️